### ISSUE TITLE
[Accessibility] Support markdown in welcome content to make it accessible. 

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#340]
Does this issue occur when all extensions are disabled?: Yes/No

- VS Code Version: 1.67.1
- OS Version: Windows 11

Steps to Reproduce:

1. In the below Live Share example . Welcome content is markdown. We would like to make the blue link text either bold or italic. But using ** doesn't take effect. it reads it as string . as a result output shows asterisk instead of a bold text. (tried with escape characters as well)  
<img width="261" alt="image" src="https://user-images.githubusercontent.com/37163639/169416959-c9b36fb8-dcc8-4c3b-8e82-d5835f8e0694.png">

**Comment by vscodenpa** [comment]
confidence for label accessibility: 0.6. does not meet threshold

**Comment by vscodenpa** [comment]
confidence for assignee chrmarti: 0.9. does meet threshold

**Comment by vscodenpa** [comment]
confidence for label accessibility: 0.6. does not meet threshold

**Comment by vscodenpa** [comment]
confidence for assignee chrmarti: 0.9. does meet threshold

**Comment by vscodenpa** [comment]
confidence for label accessibility: 0.6. does not meet threshold

**Comment by vscodenpa** [comment]
confidence for assignee chrmarti: 0.9. does meet threshold

**Comment by vscodenpa** [comment]
confidence for label accessibility: 0.6. does not meet threshold

**Comment by vscodenpa** [comment]
confidence for assignee chrmarti: 0.9. does meet threshold

**Comment by vscodenpa** [comment]
confidence for label accessibility: 0.6. does not meet threshold

**Comment by vscodenpa** [comment]
confidence for assignee chrmarti: 0.9. does meet threshold

**Comment by vscodenpa** [comment]
confidence for label accessibility: 0.6. does not meet threshold

**Comment by vscodenpa** [comment]
confidence for assignee chrmarti: 0.9. does meet threshold
